lib/cmds/initialize.ts

```diff
@@ -90,45 +90,71 @@ function setupHardwareAcceleration(sdkPath: string) {
   }
 }
 
-// Get a list of all the SDK download targets for a given set of APIs and ABIs
-function getAndroidSDKTargets(apiLevels: string[], abis: string[]): string[] {
-  return apiLevels
-      .map((level) => {
-        return 'android-' + level;
-      })
-      .concat(abis.reduce((targets, abi) => {
-        let abiParts: string[] = abi.split('/');
-        let deviceType: string = 'default';
-        let architecture: string;
-        if (abiParts.length == 1) {
-          architecture = abiParts[0];
-        } else {
-          deviceType = abiParts[0];
-          architecture = abiParts[1];
-        }
-        if (deviceType.toUpperCase() == 'DEFAULT') {
-          deviceType = 'android';
-        }
-        return targets.concat(apiLevels.map((level) => {
-          return 'sys-img-' + architecture + '-' + deviceType + '-' + level;
-        }));
-      }, []));
+// Get a list of all the SDK download targets for a given set of APIs,
+// architectures, and platforms
+function getAndroidSDKTargets(
```

Comment:
In the `initialize.ts` file, would prefer functions to be written as:
```
getAndroidSDKTargets: (apiLevels: string[], architectures: string[], platforms: string[],
  oldAVDs: string[]): string[] => {
```

over: 
```
function getAndroidSDKTargets(apiLevels: string[], architectures: string[], platforms: string[],
  oldAVDs: string[]): string[] {
```